apps/api/src/app.module.ts

```diff
@@ -103,6 +104,7 @@ const baseModules: Array<Type | DynamicModule | Promise<DynamicModule> | Forward
   TracingModule.register(packageJson.name, packageJson.version),
   BridgeModule,
   PreferencesModule,
+  WorkflowV2Module,
```

Comment:
change the old workflow module to notification template, along with the rest, mark all old workflow methods and objects as deprecated